### PR TITLE
ref(ui): Remove `grid-emotion` from a bunch of places...

### PR DESCRIPTION
**Pull request by billyvg**
Remove `grid-emotion` from a bunch of random components/views

### CODE REVIEW

File: src/sentry/static/sentry/app/views/settings/projectTags.jsx [hunk 1]
```diff
@@ -127,3 +123,19 @@ export default class ProjectTags extends AsyncView {
...
 }
+
+const TagPanelItem = styled(PanelItem)`
+  padding: 0;
+  align-items: center;
+`;
+
+const TagName = styled('div')`
+  flex: 1;
+  padding: ${space(2)};
```
Comment (dashed) [inline src/sentry/static/sentry/app/views/settings/projectTags.jsx]: Was `flex-end` not included here because it was unnecessary?

**Review by dashed — APPROVED** [review]
This must've been tedious to do. 😬 

Thanks for going through these 👍

File: src/sentry/static/sentry/app/components/group/externalIssuesList.tsx [hunk 2]
```diff
@@ -24,6 +11,20 @@ import {
...
+import ExternalIssueActions from 'app/components/group/externalIssueActions';
+import ExternalIssueStore from 'app/stores/externalIssueStore';
+import IssueSyncListElement from 'app/components/issueSyncListElement';
+import PluginActions from 'app/components/group/pluginActions';
+import SentryAppComponentsStore from 'app/stores/sentryAppComponentsStore';
+import SentryAppExternalIssueActions from 'app/components/group/sentryAppExternalIssueActions';
+import SentryAppInstallationStore from 'app/stores/sentryAppInstallationsStore';
+import SentryTypes from 'app/sentryTypes';
+import space from 'app/styles/space';
+import withOrganization from 'app/utils/withOrganization';
```
Comment (EvanPurkhiser) [inline src/sentry/static/sentry/app/components/group/externalIssuesList.tsx]: Did you get your import plugin working again?

Comment (billyvg) [reply src/sentry/static/sentry/app/components/group/externalIssuesList.tsx]: Yeah, is yours broken?

File: src/sentry/static/sentry/app/views/settings/projectTags.jsx (hunk 1 above)
Comment (billyvg) [reply src/sentry/static/sentry/app/views/settings/projectTags.jsx]: Yeah, there was/is a lot of unnecessary CSS throughout :|

**Pull request merged**